src/main/java/org/aszjch/demoapp/domain/article/ArticleServiceImpl.java

```diff
@@ -0,0 +1,73 @@
+package org.aszjch.demoapp.domain.article;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.aszjch.demoapp.domain.article.port.ArticlePublisher;
+import org.aszjch.demoapp.domain.article.port.ArticleRepository;
+import org.aszjch.demoapp.domain.articlefile.ArticleFile;
+import org.aszjch.demoapp.domain.articlefile.port.ArticleFileAssignmentService;
+
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@RequiredArgsConstructor
+public class ArticleServiceImpl implements ArticleService, ArticleFileAssignmentService {
+
+    private final ArticleRepository repository;
+    private final ArticlePublisher publisher;
+
+    @Override
+    public List<Article> get() {
```

Comment:
Zapnijmy tutaj tez Paginację i Sortowanie żeby cokolwiek się działo. Filtrowanie olejmy - zobaczymy jak wyjdzie nam przepychanie obiektów - pewnie trzeba będzie zmapować :)

Reply:
Na pewno chcemy pchać Page ze springframework? Domena chyba miała być w miarę wolna od takich rzeczy

Reply:
Tego Page to na razie odpuszczam, żeby więcej czasu na przypadki testowe poświęcić